src/Init/Data/Ord.lean

```diff
@@ -0,0 +1,39 @@
+/-
+Copyright (c) 2021 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Dany Fabian
+-/
+
+prelude
+import Init.Data.Int
+import Init.Data.String
+
+inductive Ordering where
+  | lt | eq | gt
+deriving Inhabited
+
+
+class Ord (α : Type u) where
+  compare : α → α → Ordering
+
+def cmp [inst : Ord α] := inst.compare
```

Comment:
```suggestion
export Ord (compare)
```

Reply:
i see, ok, then I'll have to change the code gen too. Will do.

Reply:
done